pkg/policy/policy.go

```diff
@@ -1,4 +1,4 @@
-package config
+package policy
 
 type PolicyWrapper struct {
```

Comment:
Not sure if policy wrapper is the best name, but lacking any better suggestion for now.

Reply:
maybe defined a type called Policies i.e 
```suggestion
type Policies []*Policy
```